msgid ""
msgstr ""
"Project-Id-Version: BackupSuite\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-07-22 19:19+0200\n"
"PO-Revision-Date: \n"
"Last-Translator: Pedro_Newbie <backupsuite@outlook.com>\n"
"Language-Team: \n"
"Language: es_ES\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Generator: Poedit 2.1.1\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../plugin.py:441
msgid ""
"\n"
"The found files:"
msgstr ""
"\n"
"Ficheros encontrados:"

#: ../plugin.py:358
msgid ""
"\n"
"You are using dual multiboot!"
msgstr ""

#: ../plugin.py:447
#, python-format
msgid "  %s (maybe error)"
msgstr "  %s (posible error)"

#: ../plugin.py:444
#, python-format
msgid "  %s (maybe ok)"
msgstr "  %s (posible ok)"

#: ../message.py:180
msgid " MB available space"
msgstr "MB de espacio disponible"

#: ../message.py:185
msgid " MB needed space"
msgstr "MB de espacio necesario"

#: ../plugin.py:450
msgid " nothing!"
msgstr " nada!"

#: ../message.py:45
msgid " size to be backed up: "
msgstr " tamaño de la copia de seguridad: "

#: ../message.py:175
msgid " to make a back-up!"
msgstr "para hacer un back-up!"

#: ../message.py:169
msgid "* * * WARNING * * *"
msgstr "* * * WARNING * * *"

#: ../plugin.py:480
msgid "A recording is currently running. Please stop the recording before trying to start a flashing."
msgstr "Una grabacion esta actualmente en marcha. Pare la grabacion antes de intentar cargar la flash."

#: ../message.py:70
msgid "Additional backup -> "
msgstr "Copia de seguridad adicional -> "

#: ../message.py:35
msgid "BACK-UP TOOL, FOR MAKING A COMPLETE BACK-UP"
msgstr "HERRAMIENTA DE BACK-UP, PARA HACER UN BACK-UP COMPLETO"

#: ../plugin.py:132
msgid "Backup > HDD"
msgstr "Backup > HDD"

#: ../plugin.py:130
#, fuzzy
msgid "Backup > MMC"
msgstr "Backup > HDD"

#: ../plugin.py:133
msgid "Backup > USB"
msgstr "Backup > USB"

#: ../plugin.py:584 ../plugin.py:591
msgid "Backup and restore your image"
msgstr ""

#: ../message.py:149
msgid "Backup done with: "
msgstr "De copia de seguridad realiza con: "

#: ../message.py:106
msgid "Backup finished and copied to your USB-flashdrive."
msgstr "Backup finalizado y copiado en su unidad USB."

#: ../plugin.py:338 ../plugin.py:541
#, fuzzy
msgid "Backup info"
msgstr "BackupSuite"

#: ../plugin.py:583 ../plugin.py:590
msgid "BackupSuite"
msgstr "BackupSuite"

#: ../plugin.py:131 ../plugin.py:233 ../plugin.py:262
msgid "Close"
msgstr ""

#: ../message.py:65
msgid "Create: kerneldump"
msgstr "Creando: kerneldump"

#: ../message.py:60
msgid "Create: root.ubifs"
msgstr "Creando: root.ubifs"

#: ../plugin.py:339 ../plugin.py:570
msgid "Delete"
msgstr ""

#: ../plugin.py:567
#, fuzzy
msgid "Delete backup"
msgstr "Restaurar backup"

#: ../plugin.py:540
#, python-format
msgid "Do you really want to unpack %s ?"
msgstr "Realmente quiere desempaquetar %s ?"

#: ../plugin.py:153
msgid ""
"Do you want to make a back-up on USB?\n"
"\n"
"This only takes a few minutes depending on the used filesystem and is fully automatic.\n"
"\n"
"Make sure you first insert an USB flash drive before you select Yes."
msgstr ""
"Quiere realizar un back-up de su imagen en el USB?\n"
"\n"
"Esto tardara pocos minutos, depende de sus sistema de archivos y es totalmente automatico.\n"
"\n"
"Asegurese primero de insertar su dispositivo USB antes de seleccionar SI."

#: ../plugin.py:150
msgid ""
"Do you want to make an USB-back-up image on HDD? \n"
"\n"
"This only takes a few minutes and is fully automatic.\n"
msgstr ""
"Quiere realizar un back-up de su imagen para USB en el disco duro? \n"
"\n"
"Esto tardara pocos minutos y es totalmente automatico.\n"

#: ../plugin.py:156
#, fuzzy
msgid ""
"Do you want to make an USB-back-up image on MMC? \n"
"\n"
"This only takes a few minutes and is fully automatic.\n"
msgstr ""
"Quiere realizar un back-up de su imagen para USB en el disco duro? \n"
"\n"
"Esto tardara pocos minutos y es totalmente automatico.\n"

#: ../plugin.py:470
msgid "Exit"
msgstr "Salir"

#: ../plugin.py:484
msgid "Flashing: "
msgstr "Flashing: "

#: ../plugin.py:369
msgid "For flashing your receiver files are needed:\n"
msgstr "Para cargar la flash el receptor se necesitan los ficheros:\n"

#: ../message.py:129
msgid "Full back-up direct to USB"
msgstr "Back-up completo directo al USB"

#: ../plugin.py:194
msgid "Full back-up on HDD"
msgstr "Back-up completo en Disco Duro"

#: ../plugin.py:208
#, fuzzy
msgid "Full back-up on MMC"
msgstr "Back-up completo en Disco Duro"

#: ../plugin.py:201
msgid "Full back-up to USB"
msgstr "Back-up completo al USB"

#: ../message.py:235
#, fuzzy
msgid "Full back-up to the MultiMediaCard"
msgstr "Back-up completo al Disco duro"

#: ../message.py:111
msgid "Full back-up to the harddisk"
msgstr "Back-up completo al Disco duro"

#: ../message.py:123
msgid "If you place an USB-stick containing this file then the back-up will be automatically made onto the USB-stick and can be used to restore the current image if necessary."
msgstr "Si usted conecta un USB-pen con este archivo, entonces el back-up se crea automaticamente en el USB-pen y puede usarlo para recuperar la imagen instalada si es necesario."

#: ../message.py:90
msgid "Image creation FAILED!"
msgstr "FALLO en la creacion de la Imagen!"

#: ../message.py:225
msgid "Installed packages contained in this backup:"
msgstr ""

#: ../message.py:154
msgid "KB per second"
msgstr "KB por segundo"

#: ../plugin.py:127
msgid "Make a backup or restore a backup"
msgstr "Realiza un backup o restaura un backup"

#: ../message.py:159
msgid "Most likely this back-up can't be restored because of it's size, it's simply too big to restore. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr "Lo más probable es que este backup no se pueda restaurar debido a su tamaño, simplemente es demasiado grande para restaurar. Esto es una limitación del bootloader.  no de la copia de seguridad o BackupSuite."

#: ../plugin.py:45
msgid "NAS"
msgstr ""

#: ../message.py:230
msgid "NFI Image created in: "
msgstr ""

#: ../plugin.py:516
msgid "NOT found files for flashing!\n"
msgstr "Ficheros para flashing no encontrados!\n"

#: ../message.py:220
msgid "No additional USB-stick found to copy an extra backup"
msgstr ""

#: ../plugin.py:173 ../message.py:30
msgid "No supported receiver found!"
msgstr "Decodificador no valido!"

#: ../message.py:170
msgid "Not enough free space on "
msgstr "Espacio insuficiente en"

#: ../plugin.py:458 ../plugin.py:499
msgid "Only kernel"
msgstr "Solo kernel"

#: ../plugin.py:457 ../plugin.py:496
msgid "Only root"
msgstr "Solo root"

#: ../message.py:118
msgid "PLEASE READ THIS:"
msgstr "POR FAVOR LEA ESTO:"

#: ../message.py:40
msgid "Please be patient, a backup will now be made, this will take about: "
msgstr "Por favor, ser paciente, a continuación se hará una copia de seguridad, esto les tomara unas: "

#: ../message.py:85
msgid "Please check the manual of the receiver on how to restore the image."
msgstr "Por favor consulte en el manual de usuario como restaurar la imagen "

#: ../plugin.py:234
msgid "Please wait"
msgstr ""

#: ../plugin.py:528
msgid "Please: DO NOT reboot your STB and turn off the power.\n"
msgstr "Por favor: NO reinicie o apague su deco.\n"

#: ../message.py:215
msgid "Probably you are trying to make the back-up in flash memory"
msgstr ""

#: ../plugin.py:134
msgid "Restore backup"
msgstr "Restaurar backup"

#: ../plugin.py:336 ../plugin.py:353 ../plugin.py:362
msgid "Run flash"
msgstr "Run flash"

#: ../plugin.py:464 ../plugin.py:511
#, fuzzy
msgid "Second partition (only kernel)"
msgstr "Estandar (root y kernel)"

#: ../plugin.py:463 ../plugin.py:508
msgid "Second partition (only root)"
msgstr ""

#: ../plugin.py:462 ../plugin.py:505
#, fuzzy
msgid "Second partition (root and kernel)"
msgstr "Estandar (root y kernel)"

#: ../plugin.py:368
msgid "Select parameter for start flash!\n"
msgstr "Elegir parametro para comenzar flash!\n"

#: ../plugin.py:261
msgid "Select the folder with backup"
msgstr "Elegir carpeta con backup"

#: ../plugin.py:521
msgid "Show only found image and mtd partitions.\n"
msgstr "Mostrar solo las imagenes y particiones mtd encontradas.\n"

#: ../plugin.py:455 ../plugin.py:487
msgid "Simulate (no write)"
msgstr "Simulacion (sin escritura)"

#: ../plugin.py:461 ../plugin.py:502
#, fuzzy
msgid "Simulate second partition (no write)"
msgstr "Simulacion (sin escritura)"

#: ../message.py:55
msgid "Some information about the task"
msgstr "Parte de la información acerca de la tarea"

#: ../plugin.py:456 ../plugin.py:490
msgid "Standard (root and kernel)"
msgstr "Estandar (root y kernel)"

#: ../message.py:134
msgid "The content of the folder is:"
msgstr "El contenido de la carpeta es:"

#: ../plugin.py:529
msgid "The image or kernel will be flashing and auto booted in few minutes.\n"
msgstr "La imagen o kernel se cargara en flash y rearrancara en pocos minutos.\n"

#: ../message.py:190
msgid "The program will abort, please try another medium with more free space to create your back-up."
msgstr "El programa abortará, utilizar otro dispositivo con mas espacio libre para crear el back-up."

#: ../message.py:124
msgid "The program will exit now."
msgstr "Saliendo ahora del programa."

#: ../message.py:164
msgid "There COULD be a problem with restoring this back-up because the size of the back-up comes close to the maximum size. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr "Podría haber un problema al restaurar esta copia de seguridad debido a que su tamaño se aproxima al tamaño máximo. Esto es una limitación bootloader, no de la copia de seguridad o BackupSuite."

#: ../message.py:116
msgid "There is NO valid USB-stick found, so I've got nothing to do."
msgstr "No se ha encontrado un USB-pen valido, No se puede hacer el backup."

#: ../message.py:100
msgid "There is a valid USB-flashdrive detected in one of the USB-ports, therefore an extra copy of the back-up image will now be copied to that USB-flashdrive."
msgstr "Se ha detectado un USB-pen valido en una de las conexiones USB, tambien tiene una copia extra del back-up de la imagen copiada en el USB-pen."

#: ../message.py:101
msgid "This only takes about 20 seconds....."
msgstr "Esto solo tarda aproximadamente 20 segundos....."

#: ../message.py:139
msgid "Time required for this process: "
msgstr "Tiempo necesario para este proceso: "

#: ../message.py:119
msgid "To back-up directly to the USB-stick, the USB-stick MUST contain a file with the name:"
msgstr "Para hacer el back-up directamente en el USB-pen, en el USB-pen debe tener el archivo con el nombre:"

#: ../message.py:75
msgid "USB Image created in: "
msgstr "Imagen para USB creada en: "

#: ../plugin.py:334 ../plugin.py:537
msgid "Unzip"
msgstr "Unzip"

#: ../message.py:200
msgid "Using your own custom lookuptable.txt from the folder /etc"
msgstr ""

#: ../message.py:205
msgid "Version unknown, probably not installed the right way."
msgstr ""

#: ../plugin.py:359
msgid ""
"Warning!\n"
"Use at your own risk! Make always a backup before use!\n"
"Don't use it if you use multiple ubi volumes in ubi layer!"
msgstr ""
"Warning!\n"
"Use bajo su responsabilidad! Realice siempre un backup antes!\n"
"No usar si utiliza multiple volumenes UBI en la capa UBI!"

#: ../plugin.py:232
msgid "What is new since the last release?"
msgstr "Qué hay de nuevo desde la última versión?"

#: ../plugin.py:571
#, python-format
msgid ""
"You are about to delete this backup:\n"
"\n"
"%s\n"
"Continue?"
msgstr ""

#: ../plugin.py:527
msgid "Your STB will freeze during the flashing process.\n"
msgstr "El deco se bloqueará durante el proceso de flasheo.\n"

#: ../message.py:80
msgid "and there is made an extra copy in: "
msgstr "ahora se esta creando una copia extra en: "

#: ../message.py:95
msgid "available "
msgstr "disponible "

#: ../message.py:120
msgid "backupstick or"
msgstr "backupstick o"

#: ../message.py:121
msgid "backupstick.txt"
msgstr "backupstick.txt"

#: ../plugin.py:266 ../plugin.py:328
#, python-format
msgid "current:  %s"
msgstr "actual:  %s"

#: ../message.py:195
msgid "is not executable..."
msgstr ""

#: ../message.py:144
msgid "minutes"
msgstr "minutos"

#: ../message.py:50
msgid "not found, the backup process will be aborted!"
msgstr "no encontrado, el backup sera abortado!"

#: ../message.py:210
msgid "not installed yet, now installing"
msgstr ""

#: ../plugin.py:526
msgid "ofgwrite will stop enigma2 now to run the flash.\n"
msgstr "ofgwrite detendra enigma2 para arrancar la flash.\n"
